scripts/scripts.js

```diff
@@ -136,5 +139,104 @@ async function loadPage() {
 // TODO: Remove once chatbot is compatible with Helix domain
 loadScript('/scripts/mockxmlhttprequest.js');
 loadPage();
-
 window.validateuserToken = '';
+
+export async function handleSocialShareClick(link) {
```

Comment:
I suggest separating the social share button from the current block, since it's under review on https://github.com/aemsites/icicidirect/pull/82. It's good to keep all comments and changes in one place.

Reply:
I had added to work PR well, will remove if #82 merge first